docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -21,6 +23,57 @@ description: >
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.19.0](https://github.com/aws/eks-anywhere/releases/tag/v0.19.0)
+
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.19.0 |    ✔    |     ✔      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
+### Added
+- Support for Kubernetes v1.29
+- Support for in-place Kubernetes version upgrade on Bare Metal clusters
+- Support for `etcd` scaling ([#7127](https://github.com/aws/eks-anywhere/pull/7127))
+- Etcd encryption for Nutanix ([#7565](https://github.com/aws/eks-anywhere/pull/7565))
+- Nutanix Cloud Controller Manager integration ([#7534](https://github.com/aws/eks-anywhere/pull/7534))
+- Enable image signing for all images used in cluster operations
+- RedHat 9 support for CloudStack ([#2842](https://github.com/aws/eks-anywhere-build-tooling/pull/2842))
+- New `upgrade management-components` command which upgrades management components independently of cluster components ([#7238](https://github.com/aws/eks-anywhere/pull/7238))
+- New `upgrade plan management-components` command which provides new release versions for the next management components upgrade ([#7447](https://github.com/aws/eks-anywhere/pull/7447))
+- Make `maxUnhealthy` count configurable for control plane and worker machines ([#7821](https://github.com/aws/eks-anywhere/pull/7281))
+
```

Comment:
Can we add below line in the `Added` section?
All the container images are signed using AWS Signer and Notation CLI.

Reply:
There's already an item for image signing in the `Added` section - `Enable image signing for all images used in cluster operations`